docs/source/developer/contributing.rst

```diff
@@ -80,51 +80,54 @@ case, please prepend ``[WIP]`` to your pull request's title.
 Build environment
 '''''''''''''''''
 
-Numba has a number of dependencies (mostly `Numpy <http://www.numpy.org/>`_
+Numba has a number of dependencies (mostly `numpy <http://www.numpy.org/>`_
 and `llvmlite <https://github.com/numba/llvmlite>`_) with non-trivial build
 instructions.  Unless you want to build those dependencies yourself, we
-recommend you use `Conda <http://conda.pydata.org/miniconda.html>`_ to
+recommend you use `conda <http://conda.pydata.org/miniconda.html>`_ to
 create a dedicated development environment and install precompiled versions
 of those dependencies there.
 
-First add the Binstar ``numba`` channel so as to get development builds of
-the llvmlite library::
+First add the Anaconda Cloud ``numba`` channel so as to get development builds
+of the llvmlite library::
 
    $ conda config --add channels numba
 
 Then create an environment with the right dependencies::
 
-   $ <path_to_miniconda>/conda create -n numbaenv python=3.5 llvmlite numpy
+   $ conda create -n numbaenv python=3.6 llvmlite numpy scipy
 
 .. note::
-   This installs an environment based on Python 3.5, but you can of course
+   This installs an environment based on Python 3.6, but you can of course
    choose another version supported by Numba.
 
 To activate the environment for the current shell session::
 
-   $ source <path_to_miniconda>/activate numbaenv
+   $ conda activate numbaenv
 
 .. note::
-   Those instructions are for a standard Linux shell.  You may need to
+   These instructions are for a standard Linux shell.  You may need to
    adapt them for other platforms.
 
 Once the environment is activated, you have a dedicated Python with the
 required dependencies::
 
-   $ python
-   Python 3.4.2 |Continuum Analytics, Inc.| (default, Oct 21 2014, 17:16:37)
-   [GCC 4.4.7 20120313 (Red Hat 4.4.7-1)] on linux
-   Type "help", "copyright", "credits" or "license" for more information.
-   >>> import llvmlite
-   >>> llvmlite.__version__
-   '0.2.0-3-g9f60cd1'
+    $ python
+    Python 3.6.6 |Anaconda, Inc.| (default, Jun 28 2018, 11:07:29)
+    [GCC 4.2.1 Compatible Clang 4.0.1 (tags/RELEASE_401/final)] on darwin
+    Type "help", "copyright", "credits" or "license" for more information.
+    >>> import llvmlite
+    >>> llvmlite.__version__
+    '0.24.0'
+
 
 Building Numba
 ''''''''''''''
 
 For a convenient development workflow, we recommend you build Numba inside
 its source checkout::
 
+   $ git clone git://github.com/numba/numba.git
+   $ cd numba
    $ python setup.py build_ext --inplace
 
 This assumes you have a working C compiler and runtime on your development
```

Comment:
The thing 2-3 lines below about re-running that command when C files are mutated... it doesn't always work.